decidim-core/app/views/decidim/notifications_settings/show.html.erb

```diff
@@ -57,6 +57,21 @@
       </div>
     <% end %>
 
+    <% if @notifications_settings.meet_push_notifications_requirements? %>
+      <div class="push-notifications">
```

Comment:
Please change the CSS so these lines are visible in mobile: 

![image](https://user-images.githubusercontent.com/717367/159431733-edb847be-5334-4273-9179-b01859310d70.png)

This is well explained at #9016